packages/react-sdk/src/arg-types/utils.ts

```diff
@@ -31,6 +31,7 @@ export const propsToArgTypes = (
     }
 
     const control = mapControlForType(prop);
+
```

Comment:
would be better to send such changes as separate PR, because this is easy to approve and merge and generates a huge diff, other things are harder to get fully done

Reply:
https://github.com/webstudio-is/webstudio-designer/pull/386
After merge/rebase that changes would gone